### PR TITLE
feat(Table View): Created table views for AllListings and OrgListings pages

### PR DESCRIPTION
**Pull request by KageRyu7**
Adds table-view as an option for management pages.

### CODE REVIEW

File: app/js/components/listing/ListingRow.jsx [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+var ProfileLink = require('../profile/ProfileLink.jsx');
+
+var ActionMenu = React.createClass({
+
+    mixins: [ Navigation, ActiveState ],
+
+    render: function () {
+
+        //TODO fill in hrefs
```
Comment (arw180) [inline app/js/components/listing/ListingRow.jsx]: is this still TODO?

File: app/js/components/CenterListingsSettings/index.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
+'use strict';
```
Comment (arw180) [inline app/js/components/CenterListingsSettings/index.jsx]: was the addition of this file a general refactor that was done during the work of this issue? it doesn't appear to be specifically related to the new table view

**Comment by bhalt** [comment]
@arw180 fixed bug where table view sort icons were white and thus didn't show up

**Comment by blaino** [comment]
Provided that we understand that this is a preliminary capability, :+1: . TODO: 1. separate grid code from normal tile view code and 2. use w2grid to match tables elsewhere in the product.

**Pull request merged**